ggml/src/ggml-sycl/ggml-sycl.cpp

```diff
@@ -3700,7 +3700,8 @@ static ggml_status ggml_backend_sycl_graph_compute(ggml_backend_t backend, ggml_
 
 #ifdef GGML_SYCL_GRAPH
     if (!g_ggml_sycl_disable_graph) {
-        if (!sycl_ctx->exec_graph && !dpct::get_device(sycl_ctx->device).has(sycl::aspect::ext_oneapi_graph)) {
+        const bool graph_support = dpct::get_device(sycl_ctx->device).has(sycl::aspect::ext_oneapi_limited_graph);
```

Comment:
I think `graph_support != limited_graph_support`. It is better to handle these two cases separately imo.

Reply:
That seems to be pretty much my question from https://github.com/ggml-org/llama.cpp/pull/12873#discussion_r2039039086, does that answer your concern?

Reply:
Oh yes.